css/index.css

```diff
@@ -56,6 +56,71 @@ html, body {
     font-family: 'Roboto', sans-serif;
 }
 
+
+.header {
+    display:flex;
+    justify-content:space-be;
+    align-items: center;
+
+}
+.header img{
+    margin:0 200px 0 200px;
+}
+.navbar{
+    display:flex;
+    margin: 50px 100px;
+    
+    }
+.button1{
```

Comment:
By styling the nav you can get everything centered on the page without having to use margins to space out the <a> elements.  By targeting nav a { } you can remove all text decoration and style all colors at one time.